samples/net/dhcpv4_client/src/main.c

```diff
@@ -57,6 +61,16 @@ static void handler(struct net_mgmt_event_callback *cb,
 	}
 }
 
+static void option_handler(const uint8_t option,
+	     struct net_dhcpv4_option_callback *cb,
+	     size_t length)
```

Comment:
nit: Please align to the opening bracket.

Reply:
Sorry about that, I forgot about it in the sample